test/chpldoc/functions/WhereClause.doc.chpl

```diff
@@ -49,7 +49,8 @@ module M {
     }
 
     // From CMO_array.chpl
-    iter these(param tag, followThis) ref where tag == iterKind.follower {
+    iter these_example(param tag, followThis) ref
```

Comment:
Is this one an issue because it is not a method?

Reply:
That was my understanding too.

Reply:
Yep. I could have just put it in a class to avoid renaming it, but as far as I can tell there's nothing special in this test about its name.